# Object target
string group_name
hb_workspace_analysis/GraspObject object
bool generate_online
---
# Result
GraspStorage grasp